book.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'person'
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] [REQUIRED] Good job so far! You should be created the many-to-many relationship between `Person` and `Book` using the intermediate class `Rental` like that:, they should be implemented similar to the following:
```ruby.rb
attr_accessor :title, :author, :rentals
```